js/map_utils.js

```diff
@@ -240,7 +240,7 @@ function updateCategory(category, prevCategory) {
   
   // update page info
   setHash('category', category);
-  documentCategorySwitch(category, prevCategory, action = "click");
+  showCategory(category, prevCategory, action = "click");
```

Comment:
not losing the event documentation because `showCategory` calls `documentCategorySwitch`